docs/spec.md

```diff
@@ -433,6 +473,38 @@ Unsubscribe from parameter updates. Only supported if the server previously decl
 }
 ```
 
+### Subscribe connection graph updates
```

Comment:
@jtbandes had a better idea to add an operation that allows to subscribe/unsubscribe to/from features. I just couldn't come up with a good name for this operation. Any proposals?

Reply:
Maybe

```json
{
  "op": "subscribeFeature",
  "feature": "connectionGraphUpdate"
}
```

But I think this non-generic version is fine too.